messenger.rst

```diff
@@ -865,6 +865,116 @@ If the message fails again, it will be re-sent back to the failure transport
 due to the normal :ref:`retry rules <messenger-retries-failures>`. Once the max
 retry has been hit, the message will be discarded permanently.
 
+Multiple failed transports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes is not enough having one single global ``failed transport`` configured because 
```

Comment:
Above this line, we need a `versionadded`. Search for that in the docs to see an example.